website/_includes/nav_footer_custom.html

```diff
@@ -0,0 +1,16 @@
+<!--
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+<!--
+Used from:
+https://github.com/just-the-docs/just-the-docs/blob/main/_includes/components/sidebar.html
+-->
+<footer class="site-footer">
+  This site uses
+  <a href="https://github.com/just-the-docs/just-the-docs">Just the Docs</a>.
+  <br />
+  Hosted on GitHub Pages.
+</footer>
```

Comment:
Could we include copyright and license info here? seems like that would have it show up on every page. Unsure if there is a custom-footer option for the pages themselves...

Reply:
There's support for custom footers (https://just-the-docs.github.io/just-the-docs/docs/customization/#custom-footer). That could actually be used instead of this; I only put it here because of the default. I worry about making long footers though, I'll check with you separately about that.

Reply:
Note, pushed with this, though still feels like a long footer to me.